src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Toggles/BitToggleTest.razor

```diff
@@ -1,18 +1,30 @@
-﻿<CascadingValue Value="Visual">
-	<BitToggle IsEnabled="@IsEnabled" 
-               @bind-IsChecked="@IsChecked" 
-               IsInlineLabel="@IsInlineLabel" 
-               OnText="@OnText" 
-               OffText="@OffText" 
-               AriaLabel="@AriaLabel">
+﻿<CascadingValue Value=Visual>
+    <BitToggle IsEnabled=IsEnabled
+               @bind-IsChecked=IsChecked
+               IsInlineLabel=IsInlineLabel
+               OnText="@OnText"
+               OffText="@OffText"
+               AriaLabel="@AriaLabel"
+               Role="@Role"
+               Label="@Label">
```

Comment:
again, Please put all the values inside "".
all of them: IsEnabled, @bind-IsChecked, and ...